postBuild

```diff
@@ -10,8 +10,6 @@ python -m pip install -e . --no-cache-dir --no-deps
 
 # lab
 jupyter labextension list
-jupyter labextension install $(cat labextensions.txt) --no-build --debug
-jupyter labextension list
```

Comment:
won't need the `jupyter lab build` either :point_down:

Reply:
613135e